### PR TITLE
2 instances of lists made into lists

### PR DESCRIPTION
**Pull request by morrowc**
Reading a list inline is janky, pulling the list content into an actual list makes it easier to find later and clear that these are N requirements.

### CODE REVIEW

File: standards/SPIFFE.md
Comment (azdagron) [inline standards/SPIFFE.md:32]: Should be consistent with the punctuation at the end of the list item (period or not).

File: standards/SPIFFE.md
Comment (azdagron) [inline standards/SPIFFE.md:37]: Same here, should be consistent with the punctuation at the end of the list item (period or not).

**Review by azdagron — COMMENTED** [review]
This looks good to me. Just a few comments.

File: standards/SPIFFE.md
Comment (morrowc) [reply standards/SPIFFE.md:37]: absolutely. done.

File: standards/SPIFFE.md
Comment (morrowc) [reply standards/SPIFFE.md:32]: done.

**Comment by azdagron** [comment]
Thanks for touching that up. One more thing; all commits into the SPIFFE organization require DCO signoff. Please see the failed check for details.

**Comment by morrowc** [comment]
ok, I think some  git history-edit/rewording/etc and DOC passes now, sweet! (and I learned something!)

**Comment by morrowc** [comment]
Do I need to assign this to a code owner?
oh, nm it is assigned ok :)

**Comment by morrowc** [comment]
Chelo..."friendly ping" and all of that stuff.

**Review by azdagron — APPROVED** [review]
Thanks!

**Pull request merged**
